README.md

```diff
@@ -1874,6 +1874,247 @@ custom_keys:
 
 <img src="https://raw.githubusercontent.com/Nerwyn/android-tv-card/main/assets/conditional_layouts.png" alt="conditional layouts example" width="500"/>
 
+
+### Example 14
+
+RGB Remote using Broadlink RM4 Pro
+
+```yaml
+type: custom:android-tv-card
+remote_id: remote.rm4_pro
+title: TV RGB
+rows:
+  - - power
+    - null
+    - poweroff
+  - - up
+    - null
+    - down
+  - - 1
+    - 2
+    - 3
+  - - 4
+    - 5
+    - 6
+  - - 7
+    - 8
+    - 9
+  - - 10
+    - 11
+    - 12
+custom_actions:
+  '1':
+    tap_action:
```

Comment:
Instead of modifying the size and background, why not use the icon `mdi:circle` and adjust it's size and color? This produces roughly the same result with less CSS:

```yaml
custom_actions:
  '1':
    icon: mdi:circle
    tap_action:
      action: call-service
      service: remote.send_command
      target:
        entity_id: remote.rm4_pro
      data:
        device: Office TV Led
        command: red
    style:
      --size: 42px
      color: red
```

Reply:
Great idea, didn't think of it when I hacked that together